pkg/registry/core/pod/rest/subresources.go

```diff
@@ -109,6 +111,10 @@ func (r *AttachREST) Destroy() {
 
 // Connect returns a handler for the pod exec proxy
 func (r *AttachREST) Connect(ctx context.Context, name string, opts runtime.Object, responder rest.Responder) (http.Handler, error) {
+	if err := ensureAuthorizedForVerb(ctx, r.Authorizer, "create"); err != nil {
```

Comment:
@liggitt  IIUIC this should be something like

```
// Pod subresources differs on the REST verbs depending on the protocol used
// SPDY uses POST that at the authz layer is translated to "create"
// Websockets uses GET that is translated to "get"
// Since the defaulting to websocket for kubectl in KEP-4006 this caused an unexpected side effect
// and in order to keep existing policies backwards compatible we always check that the "create" verb is allowed
// Ref: https://issues.k8s.io/133515
```

I prefer if we keep this commented